content/docs/06-integrations/00-external-secrets-operator.md

```diff
@@ -0,0 +1,375 @@
+          # readOnlyRootFilesystem: true
+          # runAsNonRoot: true
+          # runAsUser: 1000
+
+      resources: {}
+          # requests:
+          #   cpu: 10m
+          #   memory: 32Mi
+
+    certController:
+      # -- Specifies whether a certificate controller deployment be created.
+      create: true
+      requeueInterval: "5m"
+      replicaCount: 1
+      image:
+        repository: ghcr.io/external-secrets/external-secrets
+        pullPolicy: IfNotPresent
+        tag: ""
+      imagePullSecrets: []
+      nameOverride: ""
+      fullnameOverride: ""
+      rbac:
+      # -- Specifies whether role and rolebinding resources should be created.
+        create: true
+      serviceAccount:
+        # -- Specifies whether a service account should be created.
+        create: true
+        # -- Annotations to add to the service account.
+        annotations: {}
+        # -- The name of the service account to use.
+        # If not set and create is true, a name is generated using the fullname template.
+        name: ""
+      nodeSelector: {}
+
+      tolerations: []
+
+      affinity: {}
+
+        # -- Pod priority class name.
+      priorityClassName: ""
+
+      # -- Pod disruption budget - for more details see https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+      podDisruptionBudget:
+        enabled: false
+        minAvailable: 1
+        # maxUnavailable: 1
+
+      prometheus:
+        # -- deprecated. will be removed with 0.7.0, use serviceMonitor instead
+        enabled: false
+        service:
+          # -- deprecated. will be removed with 0.7.0, use serviceMonitor instead
+          port: 8080
+
+      serviceMonitor:
+        # -- Specifies whether to create a ServiceMonitor resource for collecting Prometheus metrics
+        enabled: false
+
+        # -- Additional labels
+        additionalLabels: {}
+
+        # --  Interval to scrape metrics
+        interval: 30s
+
+        # -- Timeout if metrics can't be retrieved in given time interval
+        scrapeTimeout: 25s
+
+        ## -- Extra environment variables to add to container.
+      extraEnv: []
+
+        ## -- Map of extra arguments to pass to container.
+      extraArgs: {}
+
+        # -- Annotations to add to Deployment
+      deploymentAnnotations: {}
+
+        # -- Annotations to add to Pod
+      podAnnotations: {}
+
+      podLabels: {}
+
+      podSecurityContext: {}
+          # fsGroup: 2000
+
+      securityContext: {}
+          # capabilities:
+          #   drop:
+          #   - ALL
+          # readOnlyRootFilesystem: true
+          # runAsNonRoot: true
+          # runAsUser: 1000
+
+      resources: {}
+          # requests:
+          #   cpu: 10m
+          #   memory: 32Mi
+```
+
+# References
+
+[Amazon IAM-Policy-Examples-ASM-Secrets](https://docs.aws.amazon.com/mediaconnect/latest/ug/iam-policy-examples-asm-secrets.html)
+
+[External Secrets](https://github.com/external-secrets/external-secrets)
+
```

Comment:
Please add this [link](https://external-secrets.io/v0.5.6/) as well in references as this is the official documentation